package.json

```diff
@@ -8746,6 +8757,11 @@
 					"group": "inline@99",
 					"alt": "gitlens.copyRemoteBranchUrl"
 				},
+				{
+					"command": "gitlens.openCurrentBranchOnRemote",
+					"when": "viewItem =~ /gitlens:branch\\b(?=.*?\\b\\+(tracking|remote)\\b)/",
+					"group": "inline@100"
+				},
```

Comment:
I'm guessing that this is a duplication issue, as it doesn't really make sense. Since if we are opening the context menu for a branch, I don't think we want an option to open the current branch.